Monopoly/Assets/Game.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Game : MonoBehaviour
+{
+    CameraMovement camera;
+    List<Player> players;
+    int numberOfTurns;
+    int numberOfPlayers;
+    int currentPlayer;
+    bool start;
+    float timeLeft;
+
+    public void SetNumberOfPlayers(int number)
+    {
+        numberOfPlayers = number;
+
+        if ( number < 2)
+            number = 2;
+        if (number > 4)
+            number = 4;
+
+        if (number == 2)
+        {
+            players.Add((Player)GameObject.Find("Teapot").GetComponent(typeof(Player)));
+            players.Add((Player)GameObject.Find("Cat").GetComponent(typeof(Player)));
+
+            Player p = (Player)GameObject.Find("Dog").GetComponent(typeof(Player));
+            p.Disable();
+            p = (Player)GameObject.Find("Hat").GetComponent(typeof(Player));
+            p.Disable();
+        }
+
+        if (number == 3)
+        {
+            players.Add((Player)GameObject.Find("Cat").GetComponent(typeof(Player)));
+            players.Add((Player)GameObject.Find("Teapot").GetComponent(typeof(Player)));
+            players.Add((Player)GameObject.Find("Dog").GetComponent(typeof(Player)));
+
+            Player p = (Player)GameObject.Find("Hat").GetComponent(typeof(Player));
+            p.Disable();
```

Comment:
co to robi?

Reply:
disable powoduje zniknięcie pionka z planszy

Reply:
to w takim razie linie 41 i 42 nie są tu zbędne?